linux/debian/control

```diff
@@ -2,7 +2,7 @@ Source: hydrogen
 Section: Multimedia
 Priority: optional
 Maintainer: Sebastian Moors <smoors@users.sf.net>
-Build-Depends: debhelper (>= 4.0.0), libsndfile1-dev, libjack-dev, libasound-dev, libtar-dev, libqt4-dev, liblrdf-dev, cmake, librubberband-dev
+Build-Depends: debhelper (>= 4.0.0), libsndfile1-dev, libtar-dev, liblrdf-dev, cmake, librubberband-dev, qt5-default, libqt5xmlpatterns5-dev, libarchive-dev, libasound2-dev, libjack-jackd2-dev, liblo-dev, libpulse-dev, libcppunit-dev, liblash-compat-dev
```

Comment:
Enforcing libjack-jackd2-dev i a bad idea. Building Hydrogen shouldn't depend on a specific version 1 or 2 of jack, then, we can put:

libjack-jackd2-dev | libjack-dev,

which will allow to build with both, preferably libjack2-dev.

PS: if I remember correctly, there is a thing in Debian which is allowing to build with libjack-jackd2-dev even if libjack-dev is asked.